packages/actor-dereference-http/package.json

```diff
@@ -41,8 +41,5 @@
     "build": "npm run build:ts && npm run build:components",
     "build:ts": "node \"../../node_modules/typescript/bin/tsc\"",
     "build:components": "componentsjs-generator"
-  },
-  "browser": {
-    "./lib/ActorRdfDereferenceHttp.js": "./lib/ActorRdfDereferenceHttp-browser.js"
```

Comment:
We still need this one here.

Reply:
Apologies, I wasn't clear enough here.
We still need the `browser` entry, but it needs to be updated to map `./lib/ActorDereferenceHttp.js` to `./lib/ActorDereferenceHttp-browser.js`.

Reply:
I'll just apply this one myself quickly...